Workbooks/UsageMetrics/Usage - Events/Usage - Events.workbook

```diff
@@ -630,7 +630,20 @@
             "timeContext": {
               "durationMs": 0
             },
+            "timeContextFromParameter": "TimeRange"
+          },
+          {
+            "id": "d7a9bf96-471f-4a31-bd08-d7f999cae0f6",
+            "version": "KqlParameterItem/1.0",
+            "name": "parseCohortEvents",
+            "type": 1,
+            "query": "print replace(@'\\\\\"', @'\\\\\\\\\"', '{UserEvent:escape}')\r\n",
```

Comment:
`print replace(@'\\\\\"', @'\\\\\\\\\"', '{UserEvent:escape}')\r\n",`

this is very concerning to me.  you're using a kusto query to replace 4x quoting of a double quote with 8x quoting of a double quote?  this seems sooooooooo strange?